### PR TITLE
Add optional "OnBranches" argument to specify which branches to clear the cache for

### PR DESCRIPTION
**Pull request by bertyhell**
See readme for more info

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,10 @@ travis-ci-cloudfront-invalidation requires six flags:
...
     p: 'TravisPullRequest'
 
+These flags are optional:
+
+    o: 'OnBranches' defaults to 'master'
+
 ## Example
 
-    travis-ci-cloudfront-invalidation -a $AWS_ACCESS_KEY -s $AWS_SECRET_KEY -c $AWS_CLOUDFRONT_DIST_ID -i '/*' -b $TRAVIS_BRANCH -p $TRAVIS_PULL_REQUEST
+    travis-ci-cloudfront-invalidation -a QDUAIUQ3OQ7I2WDNLUCO, -s f4GTRs34nRS3Ntf678lDsT87Qszvrs23lmREd57 -c Y54SDKE34MDTU1 -i '/*' -b $TRAVIS_BRANCH -p $TRAVIS_PULL_REQUEST -o 'master,stage'
```
Comment (dfillion) [inline README.md:37]: Did you just commit your own AWS credentials ? 🤔

**Review by dfillion — APPROVED** [review]
Nice, I would appreciate to have this merged and released too! Is this possible @alexpatow ?

File: README.md (hunk 1 above)
Comment (bertyhell) [reply README.md:37]: you scared me there for a second. 

But I think I randomized them a bit. it's just to help people identify what the key and secret looks like.

Comment (alexpatow) [reply README.md:37]: Ouch, glad these weren't your actual credentials!

**Comment by alexpatow** [comment]
Hey, I'm so sorry, I don't check github that often and this seems to have gotten caught up in a bunch of other notifications. I'll take a look at this ASAP.

**Comment by alexpatow** [comment]
Hi @bertyhell,

Thank you so much for creating this pull request. I like the idea, but can we make two changes before I merge it:

- [ ] Please remove the fake AWS credentials in the README you've provided. It's probably best to show people to use environment variables in the travis-ci rather than hardcoding 

- [ ] Please change line 27 in index.js to:
```const branches = (argv.OnBranches || 'master').split(',').map(branch => branch.trim());``` I think it's a bit more readable.

Please let me know if you have any questions.

Thanks,

Alex

**Comment by miczed** [comment]
Any news on this? I'm currently stuck because I cannot run the invalidation on my 'staging' branch. I can create another pull request with the changes if that's ok with you. @bertyhell

**Comment by miczed** [comment]
Wow, that was fast! Thanks 🎉

**Comment by alexpatow** [comment]
Sure! should now be published as 1.0.0. Please let me know if you run into any issues, sorry for holding you up

**Comment by miczed** [comment]
It works like a charm ;-) Thanks again!

**Pull request merged**